src/BugsnagUnity/Native/Fallback/Configuration.cs

```diff
@@ -6,7 +6,7 @@ namespace BugsnagUnity
 {
   class Configuration : AbstractConfiguration
   {
-    internal Configuration(string apiKey) : base() {
+    internal Configuration(string apiKey, bool autoNotify) : base() {
       SetupDefaults(apiKey);
```

Comment:
The value for the `autoNotify` property should be used to set the value for `AutoNotify` in this object for the non-native platforms. It is used to determine whether to capture Unity log messages.